athenet/data_loader/data_loader_buffer.py

```diff
@@ -0,0 +1,77 @@
+"""Buffer for storing large network data."""
+
+import numpy as np
+
+import theano
+
+
+class Buffer(object):
```

Comment:
Put a comment on using this class. (Loses data from 1st __setitem__ after 2nd __setitem__). Also anything else specific about this workflow, if any.